sw/device/tests/usbdev_stream_test.c

```diff
@@ -44,7 +44,7 @@
 #define TRANSFER_BYTES_VERILATOR 0x2400U
 
 // This is about the amount that we can transfer within a 1 hour 'eternal' test
-//#define TRANSFER_BYTES_LONG (0xD0U << 20)
+// #define TRANSFER_BYTES_LONG (0xD0U << 20)
```

Comment:
This seems to be unrelated, is it added on purpose?

Reply:
It was probably triggered by `clang_format`. I will revert it :)